my-webxr-app/src/testing/testLog.txt

```diff
@@ -0,0 +1,8 @@
+Triggered CSV buttonTriggered no file selected
```

Comment:
Im guessing this is the file everyone will need to add for the tester to work?

Reply:
no, this file should be there but every time you will have to manually replace this file with the downloaded results from the new test, the file that everyone will need is slightly different then this one, its just a file that contains environment variables of IS_TESTING=True and that allows us to test